tests/performance/log-results.js

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/env node
+
+/**
+ * External dependencies
+ */
+const fs = require( 'fs' );
+const path = require( 'path' );
+const https = require( 'https' );
+const [ token, branch, hash, timestamp, host ] = process.argv.slice( 2 );
+const { median } = require( './utils' );
+
+const resultsFiles = [
+	{
+		file: 'home-block-theme.test.results.json',
+		metricsPrefix: 'home-block-theme-',
+	},
+	{
+		file: 'home-classic-theme.test.results.json',
+		metricsPrefix: 'home-classic-theme-',
+	},
+];
+
+const performanceResults = resultsFiles.map( ( { file } ) =>
+	JSON.parse(
+		fs.readFileSync( path.join( __dirname, '/specs/' + file ), 'utf8' )
+	)
+);
+
+/**
+ * Gets the array or metrics.
+ *
+ * @return {array} Metrics.
+ */
+const metrics = resultsFiles.reduce(
+	( result, { metricsPrefix }, index ) => {
+		return {
+			...result,
+			...Object.fromEntries(
+				Object.entries(
+					performanceResults[ index ] ?? {}
```

Comment:
Any reason we're using this here over the (arguably more common) `||` operator?

Reply:
This function is essentially the same as what is used in the Gutenberg repo for formatting data before sending it to the API. Even so, I think that the nullish coalescing operator is appropriate to use here.

Reply:
@joemcgill Makes sense. One difference is that if `performanceResults[ index ]` was `false`, it would go through here while with `||` it rightfully wouldn't.

But that's really a theoretical concern (I don't think `performanceResults[ index ]` can ever be `false`), so not a big deal either way.

Reply:
Thank you for your feedback. I'm going to mark this conversion as resolved.